brainstormforce/ultimate-addons-for-gutenberg
# Fix: CTA - Editor padding value updated same as frontend.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/45045536-8170-4e4f-961b-323e498a6fad?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots


### Types of changes




- Updation

### How has this been tested?

- Drag and drop CTA block.
- Check button padding is same as in editor spacing control.

### Checklist:
